### PR TITLE
RSWEB-6910 Fix Hover State grid pattern

### PR DESCRIPTION
**Pull request by SamuelRandall**
Jira: https://jira.rax.io/browse/RSWEB-6910

This includes a WIP refactor of the keyoffers and business resources grid patterns.

### CODE REVIEW

File: styleguide/_themes/derek/scss/patterns/grid/businessResource.scss [hunk 1]
```diff
@@ -1,14 +1,25 @@
...
+  padding: 20px 30px 5px;
   position: relative;
+}
+
+
+.businessResource-widget-hover {
+  color: $gray-darker;
+  font-weight: 600;
+  padding-bottom: 20px;
+  text-decoration: none;
```
Comment (WidgetsBurritos) [inline styleguide/_themes/derek/scss/patterns/grid/businessResource.scss]: I think the `color`, `font-weight`, `padding-bottom` and `text-decoration` here should go under `.businessResource-widget` instead, so we're only dealing with the hover state here.

File: styleguide/_themes/derek/scss/patterns/grid/businessResource.scss [hunk 2]
```diff
@@ -29,23 +40,15 @@
...
-
-  img {
-    display: inline-block;
-    margin-right: 10px;
-    padding-top: 10px;
-    width: 60px;
+.businessResource-third {
+  @include make-xs-column(12);
+  @include make-md-column(4);
+  @include flex-box;
```
Comment (WidgetsBurritos) [inline styleguide/_themes/derek/scss/patterns/grid/businessResource.scss]: I'm not so sure using a flex box on the same class you use bootstrap columns makes a whole lot of sense. What are you trying to accomplish here?

Comment (SamuelRandall) [inline styleguide/_themes/derek/scss/patterns/grid/businessResource.scss]: We did that with ias, it doesn't do anything here but I wasn't sure why you suggested it there.

File: styleguide/_themes/derek/scss/patterns/grid/keyOffers.scss [hunk 3]
```diff
@@ -1,35 +1,63 @@
...
+.keyoffersWidget-col-full {
+  @include make-md-column(12);
+}
+
+.keyoffersWidget-header {
+  color: $brand-primary;
+}
+
+.keyoffersWidget-hover {
+  text-decoration: none;
```
Comment (WidgetsBurritos) [inline styleguide/_themes/derek/scss/patterns/grid/keyOffers.scss]: same for this text-decoration

File: styleguide/_themes/derek/scss/patterns/grid/businessResource.scss (hunk 2 above)
Comment (SamuelRandall) [inline styleguide/_themes/derek/scss/patterns/grid/businessResource.scss]: Actually, the height on the columns becomes unequal when you remove the `@include flex-box`

File: styleguide/_themes/global/scss/fonts.scss [hunk 4]
```diff
@@ -20,7 +20,7 @@ $code: Tahoma, Arial, Geneva, Helvetica;
 }
 
 @mixin copy($color: $gray-darkest) {
-  color: $gray-darkest;
+  color: $color;
```
Comment (WidgetsBurritos) [inline styleguide/_themes/global/scss/fonts.scss]: thanks for that!

Comment (SamuelRandall) [inline styleguide/_themes/global/scss/fonts.scss]: 😉

File: styleguide/_themes/derek/scss/patterns/grid/ias.scss [hunk 5]
```diff
@@ -44,6 +44,7 @@
 
 .iasWidget {
   padding: 20px 10px 0;
+  text-decoration: none;
```
Comment (SamuelRandall) [inline styleguide/_themes/derek/scss/patterns/grid/ias.scss]: Went ahead and made this change here while I was at it.

File: styleguide/_themes/derek/scss/patterns/grid/businessResource.scss (hunk 2 above)
Comment (WidgetsBurritos) [inline styleguide/_themes/derek/scss/patterns/grid/businessResource.scss]: Not entirely sure that I recommended it in the first place, but that's probably fine.

**Comment by WidgetsBurritos** [comment]
👍

**Comment by runrog** [comment]
👍 design

**Comment by bighappyface** [comment]
+1

It looks like merging this will impact existing markup on www. Are there places where `.resource` is used outside of an actual node template/process, like in a general content type?

I like the new classes and approach but want to be sure we add them in a BC way.

**Comment by WidgetsBurritos** [comment]
Closing this, because the actual commit will be pulled in via #213

**Comment by WidgetsBurritos** [comment]
Based on some changes we've decided to make to our subpanel strategy, I'm reopening this and will create a separate `www` PR for this.

**Comment by WidgetsBurritos** [comment]
Closing again in lieu of #219, since I didn't have write access to @SamuelRandall's repo.

**Pull request closed without merging**